cmd/install.go

```diff
@@ -108,14 +126,12 @@ var installCommand = &cobra.Command{
 			return fmt.Errorf("post install script failed: %s", err.Error())
 		}
 
-		fmt.Printf("🐳  Installed %s to %s\n", imageName, path.Join(pm.InstallPath, pkg.Name))
+		if hasInstall {
+			fmt.Printf("🐳  Modified %s to %s\n", installed.Image, imageName)
```

Comment:
Shouldn't it be something like `"Updated %s to use %s", path.Join(pm.InstallPath, pkg.Name), imageName` ?

Reply:
I had the same thought, but realized maybe word "updated" implies the change moves the package forward in terms of app version, when that isn't true in all cases. Unless we were to compare semver compliant tags we can't really say the package has been updated only modified.

Reply:
Understand your point. In the context, I understand it as "package", and we are actually saying the images concerned, which removes the ambiguity IMHO